components/authz-service/server/v2/project_update_manager.go

```diff
@@ -7,12 +7,13 @@ import (
 	"sync"
 	"time"
 
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
 	storage "github.com/chef/automate/components/authz-service/storage/v2"
 	"github.com/chef/automate/lib/cereal"
 	"github.com/chef/automate/lib/cereal/patterns"
 	"github.com/chef/automate/lib/logger"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
 
 	"github.com/gofrs/uuid"
```

Comment:
Sideshow comments:

1. Since you're already cleaning up the imports blocks (thanks!), let's please move these into the "third party deps" block, too.
2. What's this `uuid` thing here? If we only want v4, we should be using our `lib/uuid4` package instead.